src/io/utilities/functionSignatures.json

```diff
@@ -0,0 +1,18 @@
+{
```

Comment:
@syarra the tabcomplete function from fileexchange has nothing to do with the json files.
tabcomplete is a tool for matlab prior to 2016a which adds function signatures (and thus autocompletion) to the TC.xml file which is loaded at startup by matlab and used to assign function signatures.

@laurentheirendt, @syarra : 
The json functionSignature mechanism is a Matlab-built-in mechanism. And that mechanism relies on the functionSignaturs.json being located in the folder of the function it provides a signature for. This is actually makes sense to avoid overloading signatures (i.e. using a wrong signature because two functions have the same name, which can easily happen in the old style where everything was saved in the TC.xml)
Thus, no, they unfortunately can't be moved to a common folder.
In fact, as discussed with @laurentheirendt I would either remove the whole tabcomplete fileexchange stuff from the pull again (only keeping the jsons), or, if tabcomplete stays in, provide a utilityfunction that essentially offers to add the tabcomplete functionality for Matlab versions prior to 2016a. But if we don't decide to provide this utility function, the tabcomplete folder can be completely removed.